### PR TITLE
added a new method that will return all instance rooms

### PR DESCRIPTION
**Pull request by philipnjuguna66**
### What?
the current sdk lacks method to return all current instance rooms

### Why?key

created the method please review

----

- [x] CHANGELOG updated if relevant?

### CODE REVIEW

File: src/Chatkit.php [hunk 1]
```diff
@@ -280,6 +280,28 @@ public function createRoom($options)
...
+
+    /**
+     * @param $options
+     * @return array
+     * @throws ConfigurationException
+     * @throws ConnectionException
+     */
+    public function getAllRooms($options)
+    {
+        $queryParams = isset($options['include_private']) ?  $options['include_private']  : [ 'include_private' => true];
```
Comment (shalvah) [inline src/Chatkit.php]: Shouldn't this be:

```php
 $queryParams['include_private'] = isset($options['include_private']) ?  $options['include_private']  : true;
```

**Comment by shalvah** [comment]
Hi @philipnjuguna66, it would be nice if you added a description to your PR. Nothing elaborate, just a brief statement of why the change is needed. Also, could you update the [CHANGELOG.md](https://github.com/pusher/chatkit-server-php/blob/master/CHANGELOG.md) probably under the "Unreleased" section to reflect your change.

File: src/Chatkit.php (hunk 1 above)
Comment (philipnjuguna66) [reply src/Chatkit.php]: The idea here was to return even private rooms.

**Comment by philipnjuguna66** [comment]
@shalvah noted the above concern , I have  just updated.

**Comment by hamchapman** [comment]
Thanks a lot @philipnjuguna66 and @shalvah.

I'll merge this in now and make a new release today.

Thanks again!

**Comment by philipnjuguna66** [comment]
Welcome @hamchapman  and @shalvah

**Pull request merged**